zad1.js

```diff
@@ -0,0 +1,36 @@
+const superGra = fetch("https://raider.io/api/v1/characters/profile?region=eu&realm=burning-legion&name=Nunijo")
+const statusCheck = document.getElementById("status")
+
+superGra.then((response) => {
+    if (response.ok){
+        statusCheck.textContent = "status Promise : Fulfilled"    
+    }else{
+        statusCheck.textContent = "coś się popsuło i nie było mnie słychać"
+    }
+    return response.json()
+}, (error)=> {
+    console.log(response.ok)
```

Comment:
To nie zadziała - w tej funkcji nie masz takiego argumentu jak `response`, masz `error`.

Polecam:

`console.error(error)`